bugz/configfile.py

```diff
@@ -10,10 +10,8 @@ def load_config(UserConfig=None):
     parser = configparser.ConfigParser(default_section='default')
     DefaultConfigs = sorted(glob.glob(sys.prefix + '/share/pybugz.d/*.conf'))
     SystemConfigs = sorted(glob.glob('/etc/pybugz.d/*.conf'))
-    if UserConfig is not None:
-        UserConfig = os.path.expanduser(UserConfig)
-    else:
-        UserConfig = os.path.expanduser('~/.bugzrc')
+    UserConfig = os.path.expanduser(UserConfig) if UserConfig is not None \
+        else os.path.expanduser('~/.bugzrc')
```

Comment:
I think the original was easier to read. The main maintainer of the repo has a need for simpler to read code!